docker/frontend/Dockerfile.dev

```
@@ -0,0 +1,28 @@
+# Development Dockerfile for Mobius Frontend
+# This Dockerfile is optimized for development with hot reloading
+
+FROM node:20-alpine
+
+# Set working directory
+WORKDIR /app
+
+# Install dependencies for building native modules (if needed)
+RUN apk add --no-cache python3 make g++
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)